src/scripts.js

```diff
@@ -254,6 +282,14 @@ function displaySelectedRecipe(e) {
       addToCookbookButton.innerText = 'add to cookbook';
     }
 
+    if (currentUser.favorites.includes(selectedRecipe)) {
```

Comment:
We are aware that we have similar logic peppered in a couple places. They're not exactly the same. At this point, with little time left, we've chosen to keep it as-is since it's functional. Praying we'll have time to figure out how to make this logic more dynamic and possibly put in a helper function that will actually work. This will take some time that we don't currently have at this moment. That's the goal.